src/main/java/de/hysky/skyblocker/utils/render/gui/NonClosingPopupScreen.java

```diff
@@ -0,0 +1,78 @@
+package de.hysky.skyblocker.utils.render.gui;
+
+import net.fabricmc.api.EnvType;
+import net.fabricmc.api.Environment;
+import net.minecraft.client.gui.screen.PopupScreen;
+import net.minecraft.client.gui.screen.Screen;
+import net.minecraft.screen.ScreenTexts;
+import net.minecraft.text.Text;
+import net.minecraft.util.Identifier;
+import org.jetbrains.annotations.Nullable;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.function.Consumer;
+
+public class NonClosingPopupScreen extends PopupScreen {
+	public NonClosingPopupScreen(Screen backgroundScreen, int width, @Nullable Identifier image, Text title, Text message, List<PopupScreen.Button> buttons, @Nullable Runnable onClosed) {
+		super(backgroundScreen, width, image, title, message, buttons, onClosed);
+	}
```

Comment:
`onClosed` should be a required parameter, since this is the only difference between this and `PopupScreen`. Also, `PopupScreenWithClosing` might be a better name?